model/riscv_insts_cext.sail

```diff
@@ -230,6 +233,20 @@ mapping clause assembly = C_SRLI(shamt, rsd)
   <-> "c.srli" ^ spc() ^ creg_name(rsd) ^ sep() ^ hex_bits_6(shamt)
       if shamt != 0b000000
 
+/* ****************************************************************** */
+union clause ast = C_SRLI64 : (cregidx)
+
+mapping clause encdec_compressed = C_SRLI64(rsd)
+  <-> 0b100 @ 0b0 @ 0b00 @ rsd : cregidx @ 0b00000 @ 0b01
```

Comment:
Shouldn't this (and the other new C_x64 instructions below) have a guard on the architecture being RV128 (i.e. sizeof(xlen) == 128) similar to the guards in SHIFTIOP?

Reply:
No, they're hints in RV32/RV64.

Reply:
(the exception is RV128, where an immediate of 0 expands to `srli rd, rd, 64` etc)